src/admin/resolvers/mutations/ApprovedItem.ts

```diff
@@ -290,6 +295,11 @@ export async function importApprovedItem(
       context.db,
       data.url
     )) as ApprovedItem;
+
+    // If there's an invalid image, stop here and throw an exception
+    if (!approvedItem && e instanceof InvalidImageUrl) {
```

Comment:
nice! if we have an existing approvedItem, we can just return that - otherwise, we are trying to add a new approvedItem with invalid image, so we throw userInputError